console-extensions.json

```diff
@@ -77,5 +77,15 @@
       "perspective": "admin",
       "section": "activemq"
     }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "brokers",
+      "name": "Brokers",
+      "href": "/k8s/all-namespaces/brokers",
+      "perspective": "admin",
```

Comment:
I have a doubt on that part, why does it have to be limited to an admin perspective?

Reply:
as far i know it's depends on our requirements. so initially we added `admin` perspective.